src/useFormState.js

```diff
@@ -246,6 +253,18 @@ export default function useFormState(initialState, options) {
           touch(e);
         }
 
+        let isPristine;
+        if (isFunction(inputOptions.pristinify)) {
+          isPristine = inputOptions.pristinify(
+          formState.initialValues.get(name),
+          value,
+        );
```

Comment:
indentation is wrong

Reply:
how is that possible. i am using prettier. Perhaps, i reverted some code and did not save (and formatted) later.